config/locales/sv.yml

```diff
@@ -1740,16 +1795,30 @@ sv:
   mailers:
     login_here: "Klicka här för att gå till inloggningssidan."
     your_login_email_is: "Din e-post för att logga in är: "
-
+    log_in: &log_in Logga in
+    renewal_reqs: &renewal_requirements "Du kan förnya ditt medlemskap genom att skriva under medlemsåtagandet, ladda upp dokumentation på fortbildning samt betala medlemsavgiften."
     how_to_pay_membership: "Så här betalar du medlemsavgiften:"
     pay_membership_step: "När du är inloggad hittar du betalknappen på ditt konto."
     here_is_acct_page: "Här är en länk till ditt konto: "
 
     how_to_pay_hmarkt: "Så här betalar du H-märkespremien:"
     pay_hmarkt_step: "När du är inloggad ser du när din H-märkningspremie går ut och hur du betalar på din företagssida."
     here_is_co_page: "Här är länken där du kan redigera ditt företag: "
+    find_co_in_my_cos: Du hittar sidan för ditt företag genom att gå till menyn "Mina företag" och klicka på det företag du vill se.
     h_brand_exp_shown: "Du ser när din H-märkningspremie går ut på din företagssida."
 
+    how_to_edit_companies:
+      title: "Hur du redigerar ditt/dina företag:"
+      go_to_my_cos_menu: Väl inloggad, leta upp "Mina företag" och klicka där.
+      see_click_co_name: Du kommer att se alla företag du tillhör, listade efter namn. Klicka på det företagsnamn du vill redigera.
+      view_then_edit: Du kommer då att se information om företaget. Längst ner ser du en knapp för att redigera företaget. Klicka på den.
+      cos_you_can_edit: "Detta är de företag du kan redigera:"
+
+    how_to_upload_files: "How to Upload Files:"
+    upload_files_step1: "Log in. Go the page that shows all of your uploaded files by (1) going to your login menu. Your login menu is at the far right and says '%{login_menu_title}'."
```

Comment:
Logga in. Gå till sidan som visar alla dina uppladdade filer genom att klicka på '%{login_menu_title}'.